clang/test/Misc/warning-flags.c

```diff
@@ -18,7 +18,7 @@ This test serves two purposes:
 
 The list of warnings below should NEVER grow.  It should gradually shrink to 0.
 
-CHECK: Warnings without flags (65):
```

Comment:
The comment says we should not increase it.

Reply:
Emm, seems like increased six months ago: https://reviews.llvm.org/D154251.

Reply:
Will fix

Reply:
Maybe it's not a good example. The test is to prevent adding warnings without -W flags. I think we also need the flag here for user to mask the warning.

Reply:
agree. Will add a warning group. sorry for misleading info above.